docs/chapter_4/lesson_11.md

```diff
@@ -0,0 +1,56 @@
+Goal:
+* Learners should understand more information(params) in genesis.json
+
+# Lesson 11: Introduction to genesis.json
+
+genesis.json: Is a json file that defines the initial state of the chain, as block 0 of the chain. In this file, it will contain a lot of necessary information such as initialization time, important parameters like crisis, consensus param, bank, ...
+
+In this lesson, I will mention 3 information that is genesis_time, chain_id and consensus_param.
+
+## Guidelines
+
+1. initialize a node
+2. try to find genesis.json in ~/.baby/config. Try to look for this paragraph
+
+![client configuration](images/genesis.png)
+
+3. There are three fields that you need to care about. Explaination is already in the file.
+* genesis_time: specify how long the thread was initialized. That is, when the thread runs the build successfully, the time (UTC) will be started.
+```bash
+ "genesis_time": "2023-03-09T06:22:42.750266398Z",
+```
+
+* chain_id: is a unique identifier for your chain. (If using the same version, it helps different between two chain_id)
+```bash
+ "chain_id": "baby-1",
+```
+* consensus_param: defines consensus params.
```

Comment:
consensus_param: defines consensus params for tendermint node. This will affect how Tendermint node behaves. In most case, this should be left alone.